### PR TITLE
add idea folder to git ignore

### PR DESCRIPTION
**Pull request by Lionardo**
**This pull request fixes the following issues**:
- reduce repetitive action to add idea to git ignore

### CODE REVIEW

**Comment by intellectronica** [comment]
Why no description?

File: .gitignore [hunk 1]
```diff
@@ -31,6 +31,9 @@ env/
 # Translations
 *.mo
 
+# IDE configs
+/.idea
```
Comment (intellectronica) [inline .gitignore]: This path specification is weird, why do you need it? Wouldn't it be enough to add `*.idea`?

**Comment by Brankub** [comment]
:+1:

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**